### ISSUE TITLE
Unofficial Stronghold Patch?

### ISSUE THREAD

**Issue by cybercorpse** [Sh0wdown/UnofficialCrusaderPatch#357]
What could it be?

- Bringing some nice features from SCPatch to Stronghold classics, like healing troops and stuff;

- Modding features, like campaign editing, exploring capabilities of map editor, etc. 

Maybe this will be useful for modders. What do you think, folks?

**Comment by PitchNeeded** [comment]
Plus a skirmish mode ideally if that is possible, and of course we can create our own characters for Stronghold using AICE for the historical setting.

**Comment by cybercorpse** [comment]
> Plus a skirmish mode ideally if that is possible, and of course we can create our own characters for Stronghold using AICE for the historical setting.

Oh. That wish was one of the reasons why Stronghold Europe have started. =) BTW, I don't think it's possible. Even now, when Stronghold modding is actively progressing, a modder can only change some parameteres of the game, but he can't add something new, like a completely new unit/building/AI lord. And you're talking about a completely new mode of the game.

**Comment by Flechador** [comment]
> Plus a skirmish mode ideally if that is possible, and of course we can create our own characters for Stronghold using AICE for the historical setting

Yeah it would be really cool, imagine an skirmish mode where only Europe Units are used and also with stronghold 1 rules. Just too good to be true.

**Comment by PitchNeeded** [comment]
Yes and creating a Viking lord in a Stronghold skirmish mode would be great, would go with the setting in England and the rest of Europe during the early middle ages, and I am sure it would be possible to use the woodcutter animation to create a new Viking axemen unit.....

**Comment by Sh0wdown** [comment]
You could try applying it to the stronghold.exe, maybe some settings will work. :D

**Comment by PitchNeeded** [comment]
> You could try applying the UCP to Stronghold's executable, maybe some settings will work. :D

There is no existing skirmish mode in Stronghold though, so the settings may work but there is no way of testing it out. Would it be possible to create a skirmish mode for Stronghold?
